voorhoede/head-start
# chore: component testing

# Changes

- Adds unit tests for all components and blocks

# Associated issue

None

# How to test

1. Open project in editor
2. Run in terminal `npm ci`
3. Run `npm run test`
4. All tests should run.

# Checklist
